code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -185,7 +185,7 @@
 
 /turf/proc/handle_decompression_floor_rip()
 /turf/open/floor/handle_decompression_floor_rip(sum)
```

Comment:
Uh I think this proc should not be calling remove_tile. There is actually another proc it should be calling. remove_tile is meant for players not atmos. Is this something that monster coded? Also line above this is likely not actually needed.

Reply:
Actually line above this should have a return. I am going to investigate the blame for the potential of it being related to a exploit that I am aware of at the moment.

Reply:
```suggestion
/turf/open/floor/handle_decompression_floor_rip(sum)
	if(sum > 20 && prob(CLAMP(sum / 10, 0, 30)))
		new floor_tile(src)
		make_plating()

/turf/open/floor/engine/handle_decompression_floor_rip()
	return
```

Reply:
The above fixes a specific issue and uses the proper method that most floor interactions have for making tiles. This is a error made by monster which opened up certain unintended interactions.

Reply:
@Jalleo fix the values to not undo the PR's changes and I'll commit your suggestion

Reply:
Done. It was a mistake my end I thought I fixed it already before mentioning about it.

Reply:
Noticed a issue fixing quickly then its resolved.